### ISSUE TITLE
Update filter list on the extension popup

### ISSUE THREAD

**Issue by GKid94** [uBlockOrigin/uBlock-issues#1128]
Add an option to update filter lists on the extension browser popup

![Screenshot_1](https://user-images.githubusercontent.com/18757054/85578570-737aaf80-b63a-11ea-807b-ed98babbab7d.png)

**Comment by gwarser** [comment]
Why you need this?

**Comment by GKid94** [comment]
So we could update the filter list easily

**Comment by gwarser** [comment]
But filters are updated automatically in intervals specified by filter authors.

**Comment by GKid94** [comment]
I know but I think the interval is 12 hours, and you can update them manually quicker if they made updates to the filters

**Comment by GKid94** [comment]
But ok you don't have to add the option if you don't want to

**Comment by llacb47** [comment]
No, the interval is not "12 hours", it is specified by the filter list author. This works out fine for most people

**Comment by GKid94** [comment]
Ok

**Comment by uBlock-user** [comment]
A button that appears "active" in the popup panel when one or more filterlist is out of date, like that ?

**Comment by gwarser** [comment]
You need to explain which issues it solves, and why it's important for you. You don't even filled whole template.

**Comment by GKid94** [comment]
No like a "check for updates" button for filter lists on the popup, but you don't need to add it if you don't want to

**Comment by gorhill** [comment]
I will decline (I might have decline a similar issue in the past), the point is I don't want to make it particularly easy for majority of users to disregard the wish of filter list owners about preserving the bandwidth of the server from which their list is served.

**Comment by peace2000** [comment]
This would make updating indeed quicker. Sometimes it's necessary to update lists manually in order to check if a specific issue or unblocked ad has been fixed in newer updates. As it's not practical to wait many days for that so a manual update might be necessary. :)

I update my lists manually pretty often. I do that also because I maintain my own lists and I want Ubo to have the latest updates I have made.
